admin_tests/preprint_providers/test_views.py

```diff
@@ -150,13 +150,14 @@ def test_preprint_provider_form(self):
         formatted_rule = [[[self.parent_1._id], True]]
 
         new_data = {
+            '_id': 'newname',
             'name': 'New Name',
+            'push_share_type': 'Preprint',
```

Comment:
Naming things is hard, but I'm leaning towards `share_publish_type` instead of `push_share_type`.